cmd/models/content_block.model_test.go

```diff
@@ -0,0 +1,113 @@
+package models_test
+
+import (
+	"encoding/json"
+	"go-lms-of-pupilfirst/cmd/models"
+
+	"github.com/jinzhu/gorm/dialects/postgres"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("ContentBlock.Model", func() {
+	var (
+		course        models.Course
+		level         models.Level
+		targetGroup   models.TargetGroup
+		target        models.Target
+		targetVersion models.TargetVersion
+		contentBlock  models.ContentBlock
+	)
+
+	BeforeEach(func() {
+		// init variables
+		course = CreateCourse()
+		level = CreateLevel(course)
+		targetGroup = CreateTargetGroup(level)
+		target = CreateTarget(targetGroup)
+		targetVersion = CreateTargetVersion(target)
+		contentBlock = CreateContentBlock(targetVersion)
+
+	})
+
+	AfterEach(func() {
+		u := models.User{}
+		u.Delete()
+
+		c := models.Course{}
+		c.Delete()
+
+		l := models.Level{}
+		l.Delete()
+
+		tg := models.TargetGroup{}
+		tg.Delete()
+
+		t := models.Target{}
+		t.Delete()
+
+		tv := models.TargetVersion{}
+		tv.Delete()
+
+		cb := models.ContentBlock{}
+		cb.Delete()
```

Comment:
Small comment: you can probably factor this out into a utility method, since you're doing it in quite a few places.